lighthouse-viewer/test/fileuploader-test.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* eslint-env mocha */
+
+const assert = require('assert');
+
+const testHelpers = require('./test-helpers'); // before other src imports.
+const FileUploader = require('../app/src/fileuploader');
+const logger = require('../app/src/logger');
+
+function assertUIReset(uploader) {
+  assert.ok(!logger.el.classList.contains('show'));
```

Comment:
The logger module creates a singleton to a DOM node and FileUploader uses `logger` internally to the module. It's a bit crude, but the intent here is a test that checks the state of the DOM. We don't really have another way to check that UI is working as it should.